gpiozero/boards.py

```diff
@@ -1260,11 +1263,19 @@ def decimal_point(self):
         if len(self) > 7:
             return self[7].value 
         else:
-            raise OutputDeviceError('There is no 8th pin for the decimal point')
+            raise OutputDeviceError('there is no 8th pin for the decimal point')
     
     @decimal_point.setter
     def decimal_point(self, value):
         if len(self) > 7:
             self[7].value = value
         else:
-            raise OutputDeviceError('There is no 8th pin for the decimal point')
\ No newline at end of file
+            raise OutputDeviceError('there is no 8th pin for the decimal point')
+            
+    def set_char_layout(self, char, layout):
```

Comment:
I purposely made `._layouts' private to the instance rather than static as I felt it might be confusing if a change was reflected across multi instances.